projects/tuist/fixtures/ios_app_with_static_frameworks/Prebuilt/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-swift run tuist generate
```

Comment:
is this change desirable? This will run tuist version on the system which might not be always desirable (e.g. there might be a lot of changes between latest released tuist and the version the PR builds this for (think 2.0 version))

Reply:
Cc: @kwridan, can we revert this one then?

Reply:
I accidentally checked this in, we can revert it. 

The reason why I needed to change it, with the current repository setup where there are several Package.swift files in various directories SwiftPM is unable to locate the root tuist package to build and run it should you invoke build.sh from this directory.